src/frontend/context/TidslinjeContext.ts

```diff
@@ -0,0 +1,120 @@
+import createUseContext from 'constate';
+import { SyntheticEvent, useState } from 'react';
+import { Skalaetikett } from '@navikt/helse-frontend-tidslinje/lib/components/types.internal';
+import moment from 'moment';
+import { ToggleKnappPureProps } from 'nav-frontend-toggle';
+import { IPersonBeregning, IYtelsePeriode } from '../typer/beregning';
+import { Periode } from '@navikt/helse-frontend-tidslinje/lib';
+
+export interface ITidslinjeSkala {
+    id: number;
+    navn: string;
+    måneder: number;
+}
+
+export enum TidslinjeSkala {
+    HALVT_ÅR,
+    ETT_ÅR,
+    TRE_ÅR,
+}
+
+const [TidslinjeProvider, useTidslinje] = createUseContext(() => {
+    const tidslinjeSkalaer: ITidslinjeSkala[] = [
+        { id: TidslinjeSkala.HALVT_ÅR, navn: '6 mnd', måneder: 6 },
+        { id: TidslinjeSkala.ETT_ÅR, navn: '1 år', måneder: 12 },
+        { id: TidslinjeSkala.TRE_ÅR, navn: '3 år', måneder: 36 },
+    ];
+
+    const [aktivEtikett, settAktivEtikett] = useState<Skalaetikett | undefined>(undefined);
+
+    const [tidslinjeInput, settTidslinjeInput] = useState({
+        aktivSkala: tidslinjeSkalaer[1],
```

Comment:
Kanskje man kan bruke .find hvor id = TidslinjeSkala.Ett_ÅR for å gjøre initverdi tydeligere?

Reply:
Hva med tidslinjeSkalaer[TidslinjeSkala[ETT_ÅR]? Hvis jeg bruker find kan det potensielt bli undefined, og her "vet" vi jo at den ikke skal være undefined noen gang 🤔

Reply:
gøres bra ut :D